addons/binding/org.openhab.binding.network/ESH-INF/i18n/network_de.properties

```diff
@@ -1,46 +1,46 @@
 # binding
 binding.network.name = Network Binding
-binding.network.description = Das Network Binding überprüft, ob sich ein Gerät aktuell im Netzwerk befindet oder nicht.
+binding.network.description = Das Network Binding �berpr�ft, ob sich ein Ger�t aktuell im Netzwerk befindet oder nicht.
 binding.config.network.allow_system_pings.label = Erlaubt System Pings
-binding.config.network.allow_system_pings.description = Nutzt das Ping Programm des Systems zusätzlich zum Java ping.
+binding.config.network.allow_system_pings.description = Nutzt das Ping Programm des Systems zus�tzlich zum Java ping.
 binding.config.network.allow_dhcp_listen.label = Erlaubt DHCP Sniffing
-binding.config.network.allow_dhcp_listen.description = Lauscht auf DHCP Packete, welche beim Eintritt von Geräten in das Network gesendet werden, um die Verfügbarkeit eines Gerätes in beinahe Echtzeit mitzuteilen.
+binding.config.network.allow_dhcp_listen.description = Lauscht auf DHCP Packete, welche beim Eintritt von Ger�ten in das Network gesendet werden, um die Verf�gbarkeit eines Ger�tes in beinahe Echtzeit mitzuteilen.
```

Comment:
Small typo here: "Packete"

Reply:
@claell I will probably not be available for some days in some days, would be awesome if we don't block this PR by i18n. This is not part of this PR, I just correct the encoding problem here.